pd/src/db/validators.sql

```diff
@@ -1,6 +1,5 @@
 CREATE TABLE IF NOT EXISTS validators (
     tm_pubkey bytea NOT NULL PRIMARY KEY,
     voting_power bigint NOT NULL,
-    commission_address varchar NOT NULL,
-    commission_rate bigint NOT NULL
+    funding_streams bytea NOT NULL
```

Comment:
Rather than storing the funding streams opaquely, could we keep a first-class table in the database, so we can query it later if we want to do other stuff with it?  I'm not sure what the right way to structure the table would be, since I guess we want a many-to-one relationship between funding streams and validators (cc @aubrey-dyer)

Reply:
simplest way might just be adding a funding streams table with a key or indexed column that points back to the associated validator